source/administration/config-settings.rst

```diff
@@ -620,6 +620,35 @@ Maximum file size for message attachments entered in megabytes in the System Con
 .. note::
   If you use a proxy or load balancer in front of Mattermost its settings need to be adjusted accordingly. For NGINX use ``client_max_body_size``. For Apache use ``LimitRequestBody``.
 
+Enable Document Search by Content
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Document content search is available in Mattermost Server from v5.35, with mobile support coming soon. Searching document contents adds load to your server. For large deployments, or teams that share many large, text-heavy documents, we recommended you review our `hardware requirements <https://docs.mattermost.com/install/requirements.html#hardware-requirements>`__, and test enabling this feature in a staging environment before enabling it in a production environment.
```

Comment:
from v5.35 > v5.35 and later

Reply:
@esethna This is the convention we use throughout the product documentation today to indicate when something is available in a given release or later. The "or later" is implied rather than stated.